test/jdk/java/foreign/RenderTest.java

```diff
@@ -0,0 +1,375 @@
+/*
+ * Copyright (c) 2020, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
new test. why 2020?

Reply:
The test class was copied from an existing test class. I will fix it.